confgenerator/testdata/valid/linux/default_config/golden_otel.conf

```diff
@@ -35,6 +35,8 @@ processors:
       exclude:
         match_type: strict
         metric_names:
+          # Temporarily exclude system.cpu.time (cpu/usage_time)
+          - system.cpu.time
```

Comment:
Have you built the agent and test out this config?
As in line 47, system.cpu.time is converted in to cpu/usage_time. I am concerned if it will produce error msgs, as it will be converting something that doesn't exist

Reply:
Just tested, it won't cause any warning/error msgs.